src/web/routes/application/paths/prefix-path.js

```diff
@@ -0,0 +1,23 @@
+const { isUndefined, isString } = require('../../../../common/predicates')
+
+const isInvalidSegment = segment => !isString(segment) || !segment.startsWith('/')
+
+const prefixPath = (prefix, path) => {
+  if (isInvalidSegment(path)) {
+    throw new Error(`Invalid path "${path}". Path must be a string starting with "/"`)
+  }
+
+  if (isUndefined(prefix)) {
```

Comment:
Is this check necessary? Won't it be covered by isInvalidSegment(prefix) below?

Reply:
No, because this returns the path unchanged. isInvalidSegment throws an error.